src/main/scala/mastermind/Mastermind.scala

```diff
@@ -10,22 +10,31 @@ import scala.io.StdIn.readLine
 type Color = Char
 type Board = String
 val validColors = List('B', 'Y', 'R', 'G')
+val random = new Random()
 
 /** Get a random color from the list of valid colors */
 def getRandomColor(): Color =
-  ???
+  validColors.apply(random.nextInt(4))
```

Comment:
Interesting way of solving I like your use of validColors.apply. Instead of using a loose 4 in your code maybe use the following:
```suggestion
  validColors.apply(random.nextInt(validColors.length))
```